### ISSUE TITLE
Ignite Bug (Fire Mage)

### ISSUE THREAD

**Issue by Sephix2005** [SunwellTracker/issues#3110]
Decription: When u crit a fireball with an instant Pyroblast casted right after the fireball, it counts only one crit for the damages of the generated ignite

How it works: When u crit a fireball with an instant Pyroblast casted right after the fireball, it counts only one crit for the damages of the generated ignite

How it should work: Both Crits should be added to the current damages left on the ignite (on the boss)

Source (you should point out proofs of your report, please give us some source): Fire mage were top tier back then on retail; because of this bug, they're way below arcane mages. Big issue then

**Comment by Rosenkrantz** [comment]
TBH this bug always existed on retail (see: https://wowwiki.fandom.com/wiki/Ignite_(old) known bugs and ignite munching), and Mages were topping DPS with this bug, not because it didn't exist. If it were to be fixed, it would probably require some custom nerfs to fire mages to prevent them from overperforming.
You absolutely sure this is bugged more than it should?

**Comment by Soufly** [comment]
^,most of the mages were frostfire/arcane up to ICC.

**Comment by Sephix2005** [comment]
Feels just weird, critting 9k +10k, resulting in an ignite of 1k3 per tick
reproduced it again with two 8k crits, getting 1k6 ignite

It feels just bad, and maybe i'll investigate more on this because it doesnt feel like when i was playing in retail

I know it happened sometimes on retail, but i couldn't reproduce it that easily
maybe latency makes the difference? 
Idk

**Comment by Soufly** [comment]
''Under the Fire tree you'll see a talent called Ignite which applies a DoT to the target after it receives a critical from a fire spell. The DoT is 40% of the spells damage over 4 seconds, but here's the important thing....

If you crit again within those 4 seconds while Ignite is active it'll reset the duration AND stack the new spell onto the DoT. What's even sicker is that all Mages can contribute to a single Ignite effect - that's why you see videos of Ignite dealing thousands of damage per tick.''

**Comment by Rosenkrantz** [comment]
> If you crit again within those 4 seconds while Ignite is active it'll reset the duration AND stack the new spell onto the DoT. What's even sicker is that all Mages can contribute to a single Ignite effect - that's why you see videos of Ignite dealing thousands of damage per tick.''

That's pre-TBC stuff.

**Comment by Soufly** [comment]
Yeah some random retail comment,didnt really check the date.

**Comment by Sephix2005** [comment]
Yes, later ignite works exactly as stated on the pre-TBC stuff, except it is one ignite for each fire mage (separated) but this ignite bug prevent it from working properly, and it hurts a lot the damages of firemages because it’s 40% of the fireball crit that just disappear
It happened on retail, but it was not that easily reproductible 
Here it happens every time while doing the optimal rotation (same one as back then in retail)

**Comment by Soufly** [comment]
https://www.mmo-champion.com/threads/861548-Ignite-Munching-What-Happened ,some cata comments.

**Comment by Sephix2005** [comment]
Looks like there is indeed a problem on ignite here then, according to these comments; even if it happened in retail it was quite rare

On sunwell, you can reproduce it all along your rotation, fireball+pyroblast crits, or when living bomb crit with fireball, etc it is so common that it nerfs highly ignites damages.

Now, the question I ask to myself, is it due to the high lag we are getting on the server these days? Or is there a way to fix it?

**Comment by Soufly** [comment]
https://github.com/SunwellTracker/issues/issues/3323
No.
